relentless/simulate/dilute.py

```diff
@@ -86,13 +86,10 @@ def __call__(self, sim):
         ens.clear()
 
         # pair distribution function
-        for pair in sim.potentials:
-            r = sim.potentials[pair].get('r')
-            u = sim.potentials[pair].get('u')
-            if r is None or u is None:
-                raise ValueError('r and u must be set in the potentials matrix.')
-            gr = np.exp(-sim.ensemble.beta*u)
-            ens.rdf[pair] = RDF(r,gr)
+        for pair in ens.rdf:
```

Comment:
I tweaked the `Ensemble` object to give it a `pairs` property as well:
```suggestion
        for pair in ens.pairs:
```